ENIGMAsystem/SHELL/Graphics_Systems/Direct3D9/Direct3D9Headers.h

```diff
@@ -19,20 +19,16 @@
 #define ENIGMA_DIRECT3D9_HEADERS_H
 
 #include <map>
-#include <string>
-#include <cmath>
+#include <cstring> // std::memcmp
 
 #include <d3d9.h>
-#include <windows.h>
-#include <windowsx.h>
 
 using namespace std;
 
 using std::map;
 
 namespace enigma {
 
-extern HWND hWnd;
```

Comment:
The window handle was moved to the bridge because there was no need to expose it to all of D3D9.